wright/src/grammar/parsers/expression/identifier.rs

```diff
@@ -9,7 +11,7 @@ use nom::IResult;
 impl<'s> Identifier<'s> {
     /// Reserved words that an identifier must not match.
     pub const RESERVED_WORDS: [&'static str; 3] =
-        [BooleanLit::FALSE, BooleanLit::TRUE, Underscore::UNDERSCORE];
+        [BooleanLit::FALSE, BooleanLit::TRUE, SelfLit::SELF];
```

Comment:
Even if `_` is a pattern and not an identifier, it should probably still be considered a reserved word since it has special meaning.

Reply:
For example `let _ = a;`.

Reply:
Done!